packages/pilot/src/containers/Capture/Form/index.js

```diff
@@ -0,0 +1,228 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import {
+  Alert,
+  Button,
+  Col,
+  Flexbox,
+  FormInput,
+  Grid,
+  ModalActions,
+  ModalContent,
+  Row,
+  Spacing,
+} from 'former-kit'
+import Form from 'react-vanilla-form'
+import WarningIcon from 'emblematic-icons/svg/Warning32.svg'
+
+import Property from '../../../components/Property'
+import currency from '../../../formatters/currency'
+import CaptureDetails from '../../../components/CaptureDetails'
+import CurrencyInput from '../../../components/CurrencyInput'
+import formatCardNumber from '../../../formatters/cardNumber'
+import numberValidation from '../../../validation/number'
+import requiredValidation from '../../../validation/required'
+import greaterThanValidation from '../../../validation/greaterThan'
+import lessThanOrEqualValidation from '../../../validation/lessThanOrEqual'
+
+import style from './style.css'
+
+const isRequired = t => requiredValidation(t('pages.required_error'))
+const isNumber = t => numberValidation(t('pages.capture.number'))
+
+const greaterThanAuthorized = (authorizedAmount, t) =>
+  greaterThanValidation(authorizedAmount, t('pages.capture.invalid_amount'))
+const lessThanOrEqualZero = t =>
+  lessThanOrEqualValidation(0, t('pages.capture.greater_than_zero'))
+
+const CaptureForm = ({
+  authorizedAmount,
+  cardBrand,
+  cardFirstDigits,
+  cardLastDigits,
+  captureAmount,
+  customerName,
+  customerEmail,
+  disabled,
+  isFromCheckout,
+  installments,
+  onConfirm,
+  t,
+}) => {
+  const labels = {
+    amount: t('pages.transaction.header.card_amount'),
+    captureAmount: t('pages.capture.value_to_capture'),
+    cardBrand: t('models.card.brand'),
+    cardNumber: t('models.card.number'),
+    customerName: t('models.customer.name'),
+    customerEmail: t('models.customer.email'),
+    installments: t('installments'),
+  }
+
+  const contents = {
+    cardBrand,
+    cardNumber: `${formatCardNumber(cardFirstDigits)} ${cardLastDigits}`,
```

Comment:
aqui precisa alguma validação pra saber ser o cardFirstDigits é valido, atualmente ele formata o valor mesmo que o codigo seja invalido. Dá de fazer algo mais bonito que o ssnipt abaixo, é só pra ilustrar mesmo 

```suggestion
     cardNumber: `cardFirstDigits ? `${formatCardNumber(cardFirstDigits)} ${cardLastDigits}` : null,
```

Reply:
Boa! Como não encontrei validações de número de cartão, o que fiz foi validar as duas variáveis `cardFirstDigits` e `cardLastDigits` para garantir, o que acha?